NeuralNetworkBatch states
---
float32 loss
